airbyte-cdk/python/airbyte_cdk/sources/streams/core.py

```diff
@@ -25,9 +27,9 @@
 # A stream's read method can return one of the following types:
 # Mapping[str, Any]: The content of an AirbyteRecordMessage
 # AirbyteMessage: An AirbyteMessage. Could be of any type
-StreamData = Union[Mapping[str, Any], AirbyteMessage]
 
 JsonSchema = Mapping[str, Any]
+StreamData = StreamData
```

Comment:
I moved the Type to `types.py` to avoid a circular dependency, but `StreamData` is imported from this file in a few connectors. I think removing this type would be a breaking change...